testutils/cmds/compile_solidity.go

```diff
@@ -0,0 +1,134 @@
+//go:build none
+//+build none
+
+// Copyright (c) 2020 The qitmeer developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+)
+
+// npm install solcjs
+// solcjs--version
+// 0.8.3+commit.8d00100c.Emscripten.clang
+
+const SOLC = "solcjs"
+
+// go run github.com/Qitmeer/go-etherum/cmd/abigen/main.go
+const ABIGEN = "abigen"
+
+var fileContent = "// It is called by go generate and used to automatically generate pre-computed \n// Copyright 2017-2022 The qitmeer developers \n// This file is auto generate by : go run compile_solidity.go \npackage testutils\n\n"
+
+func main() {
```

Comment:
Can we add environment detection of SOLC and ABIGEN in github workflows, and support `go run complile_solidity.go`   in Makefile?